examples/simple_example/variables.tf

```diff
@@ -28,3 +28,8 @@ variable "prefix" {
   description = "Prefix used to generate bueckt names."
   type        = "string"
 }
+
+variable "bucket_policy_only" {
```

Comment:
No need to make this a variable in the example IMO. Let's just leave it off.

Reply:
To be clear -- leave out test coverage for this functionality?

Reply:
I see now that the way tests are set up in this repo is (slightly) suboptimal, so we can leave as-is.